client/post-editor/editor-revisions-list/style.scss

```diff
@@ -59,6 +82,7 @@
 }
 
 // Individual Revision List Item
+.editor-revisions-list.is-loading .editor-revisions-list__list:before,
```

Comment:
This seems to be breaking the list item styles...

* selected items aren't highlighted
* hover effects aren't working, etc.

Basically, all of [this](https://github.com/Automattic/wp-calypso/blob/1694c137112086ab23082e8e638e120b0885724e/client/post-editor/editor-revisions-list/style.scss#L87-L116) seems to not be hitting the selector

Reply:
Thanks for pointing this out, looking at calypso.live I see it as well. I'm very certain this was not the case when working on it locally. Looking into it right now 🤔

Reply:
While the selector is correct, it looks like the non-debug build slips up on the nested selectors further down. So I moved the second part of the styles of `.editor-revisions-list__button` into its own selector and that does the trick.